tests/precompile/solidity/suites.go

```diff
@@ -6,13 +6,106 @@ package solidity
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
 	"time"
 
+	"github.com/ava-labs/avalanchego/api/health"
 	"github.com/ava-labs/subnet-evm/tests/utils"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/go-cmd/cmd"
 	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
 )
 
-var _ = ginkgo.Describe("[Precompiles]", ginkgo.Ordered, func() {
+// Timeout to boot the AvalancheGo node
+var bootAvalancheNodeTimeout = 5 * time.Minute
+
+// Timeout for the health API to check the AvalancheGo is ready
+var healthCheckTimeout = 5 * time.Second
+
+// At boot time subnets are created, one for each test suite. This global
+// variable has all the subnets IDs that can be used.
+//
+// One process creates the AvalancheGo node and all the subnets, and these
+// subnets IDs are passed to all other processes and stored in this global
+// variable
+var subnetsIds map[string]string
+
+// Keep track of the AvalancheGo external bash script, it is null for most
+// processes except the first process that starts AvalancheGo
+var startCmd *cmd.Cmd
```

Comment:
Could we put this inside one var block and update `subnetsIds` to `subnetIDs` ?

Unfortunately, we don't have a style guide for this stuff 😅

happy to make a follow on PR if you prefer